tracing/span_test.go

```diff
@@ -11,21 +11,21 @@ import (
 )
 
 func TestDebugFlag(t *testing.T) {
-	var span Span
+	span := CreateServerSpan(nil, "test")
 
 	t.Run(
 		"set",
 		func(t *testing.T) {
 			span.SetDebug(true)
-			if span.flags != FlagMaskDebug {
+			if span.trace.flags != FlagMaskDebug {
```

Comment:
🔕 A few of those tests probably should also be moved into `trace_test.go`, but I don't feel strongly about that.